uwp/Add-Syncfusion-Controls.md

```diff
@@ -25,12 +25,12 @@ Syncfusion UI for UWP are added automatically to the Visual Studio Toolbox durin
 
 2. Find `SfTextBoxExt` by typing the text “SfTextBoxExt” in search box.
 
-![Search SfTextBoxExt](Add-Syncfusion-Controls_images/Add-Syncfusion-Controls_img1.jpeg)
+![Search SfTextBoxExt in UWP](add-syncfusion-controls_images/uwp-add-syncfusion-controls-search-sftextboxext.jpeg)
```

Comment:
Change the image name like,
search-textbox-in-uwp-controls-list

Reply:
I have changed the image name